### PR TITLE
EDGAR release 22.0.1 updates

### PR DESCRIPTION
**Pull request by derekgengenbacher-wf**
#### Descripton: 
Edgar Release 22.0.1 updates

#### Testing:
Make sure that there are no regressions and no security holes

**review**:
@hermfischer-wf
cc: @austinmatherne-wk @derekgengenbacher-wf @sagesmith-wf

### CODE REVIEW

File: Filing.py [hunk 1]
```diff
@@ -184,6 +184,12 @@ def __init__(self, controller, modelXbrl, outputFolderName):
         self.srtNamespace = next((n for n in self.modelXbrl.namespaceDocs.keys() if n is not None and re.search('fasb.org/srt/20',n) is not None),None)
         self.isRR = self.rrNamespace is not None
         self.isInvestTaxonomyInDTS = self.investNamespace is not None
+        self.isN2Prospectus = not self.isRR and any(
+            f.xValue in {'N-2', 'N-2/A', 'N-2ASR', 'N-2MEF', 'N-2 POSASR', '497'}
+            for f in self.modelXbrl.factsByLocalName["DocumentType"])
+        self.isProspectus = bool(next((n for n in self.modelXbrl.namespaceDocs.keys() if n is not None and re.search('sec.gov/vip',n) is not None),None))
```
Comment (sagesmith-wf) [inline Filing.py:190]: Nit: spaces after also applies to next line commas

Comment (sagesmith-wf) [inline Filing.py:190]: Nit: Suggest a function that takes an str and a map to search with

File: Summary.py [hunk 2]
```diff
@@ -95,9 +95,10 @@ def __init__(self, controller):
         self.footnotesReported = 0 < sum([s.footnoteCount for s in summaries])
         self.scenarioCount = sum([s.scenarioCount for s in summaries])
         self.hasRR = next((True for s in summaries if s.hasRR), False)
-        self.namespacesInUse = set()
+        self.namespacesFactsCount = {}
```
Comment (sagesmith-wf) [inline Summary.py:98]: nit: suggest default dict with default value of 0 to simplify puts

**Review by derekgengenbacher-wf — COMMENTED** [review]
+1

**Pull request merged**